### PR TITLE
Fix/847 arc pm crashing

### PR DESCRIPTION
**Pull request by seanbrookes**
connected to strongloop-internal/scrum-nodeops#847

### CODE REVIEW

File: client/www/scripts/modules/pm/pm.services.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ PM.service('PMAppService', [
           return response;
         })
         .catch(function(error) {
-          $log.error('bad deploy local app: ' + error.message);
+          $log.error('bad deploy local app: ' + JSON.stringify(error));
```
Comment (kraman) [inline client/www/scripts/modules/pm/pm.services.js]: Error should have a `message` field. Is that missing? Is it a real Error object?

**Comment by kraman** [comment]
One question, rest LGTM

File: client/www/scripts/modules/pm/pm.services.js (hunk 1 above)
Comment (seanbrookes) [inline client/www/scripts/modules/pm/pm.services.js]: seems hit and miss.  I'm just stringifying it to avoid exceptions

**Comment by seanbrookes** [comment]
test please

**Comment by anthonyettinger** [comment]
test please

**Pull request merged**